.github/workflows/template.queryevents.yml

```diff
@@ -0,0 +1,65 @@
+name: "Template-Query-Event"
+
+on:
+  workflow_call:
+    outputs:
+      events:
+        description: "Json array with events"
+        value: ${{ jobs.QueryEvent.outputs.events}}
+    inputs:
+      pipelineName:
+        type: string
+        required: true
+      eventName:
+        type: string
+        required: true
+      eventGroupId:
+        type: string
+        required: true
+      keyVaultName:
+        type: string
+        required: true
+    secrets:
+      azureEnviromentCredentials:
+        required: true
+
+jobs:
+  QueryEvent:
+    name: Query Events
+    runs-on: ubuntu-latest
+    outputs:
+      events: ${{ steps.QueryEvents.outputs.events }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: Azure/login@v1
+        name: Run Azure Login
+        with:
+          creds: "${{ secrets.azureEnviromentCredentials }}"
+          environment: "AzureCloud"
+
+      - name: Get KeyVault Secret(s)
+        id: LoadSecrets
+        env:
+          SECRETS: "clientId, clientSecret, subscriptionId, tenantId, eventsStorageAccount, eventsTableName"
+          KEY_VAULT_NAME: ${{ inputs.keyVaultName }}
+        run: ./github.loadsecrets.sh
+        working-directory: scripts/orchestrators
+
+      - name: Query Events
+        id: QueryEvents
+        env:
+          WORKSPACE_PATH: ${{ github.workspace }}
+        run: |
+          export ARM_TENANT_ID=${{ steps.LoadSecrets.outputs.tenantId }}
```

Comment:
I  would use env variables for the ARM_ variables instead of export.

Reply:
Just tried to change it and had to roll the change back. Currently, the script that pulls the secrets is retrieving the values from a json and it's keeping the surrounding double-quotes. Changing that so it can be properly used in the ENV section would be simple, but it would also break other workflows depending on those double-quotes.

For now, I'll leave it as is, and we can remove the double-quotes and unify it in ENV later.